lib/rcfpch/rcf_pch_internal.h

```diff
@@ -22,8 +22,15 @@
 #include "rcf_ch_api.h"
```

Comment:
LGTM, but I guess you forget to specify Issue which the changeset addresses. Please, see examples of issue referencing in history.

Reply:
This fix and https://github.com/oktetlabs/test-environment/issues/60 issue are not connected.